packages/api-server/src/plugins/graphql.ts

```diff
@@ -9,56 +9,54 @@ import type {
 import fastifyRawBody from 'fastify-raw-body'
 import type { Plugin } from 'graphql-yoga'
 
+import type { GlobalContext } from '@redwoodjs/context'
+import { getAsyncStoreInstance } from '@redwoodjs/context/dist/store'
+// @ts-expect-error TODO(jtoar)
+import { coerceRootPath } from '@redwoodjs/fastify-web/helpers'
```

Comment:
Could use some help with this one; not sure why TS doesn't seem to recognize the subpath imports

Reply:
Try making it a deep /dist import instead